apps/web/src/app/pages/trade/xswap/_components/XSwapModal.tsx

```diff
@@ -121,13 +124,23 @@ const XSwapModal = ({ account, currencies, executionTrade, direction, recipient,
 
   const gasChecker = useXCallGasChecker(direction.from);
 
+  const ethereumChainId = useChainId();
```

Comment:
use useAccount's chainId

Reply:
it doesn't work, the value is undefinend